core/lib/web3_decl/src/client/mod.rs

```diff
@@ -196,7 +196,7 @@ impl<Net: Network, C: ClientBase> Client<Net, C> {
             origin,
             &stats,
         );
-        tracing::warn!(
+        tracing::debug!(
```

Comment:
@tomg10 You were previously advocating for the WARN level here. The thing is: as currently implemented (i.e., when this is used as the only rate limiter), this will spam a lot of warnings when a node is catching up using JSON-RPC communication. Hence, this change.

Reply:
I wanted to have it as warn as I don't think any component should rely on the fact that there exists a rate limitter, i.e. be implemented/configured in a way that doesn't send too many requests

Reply:
for instance as this may throttle other components as far as I remember